### PR TITLE
DOCSUP-2807: Documented the date_trunc function

### PR DESCRIPTION
**Pull request by olgarev**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category:
- Documentation for #13888

### CODE REVIEW

**Comment by ucasFL** [comment]
In fact, function `now` currently can accept an optinal argument: timezone.[#15285](https://github.com/ClickHouse/ClickHouse/pull/15285)

File: docs/en/sql-reference/functions/date-time-functions.md [hunk 1]
```diff
@@ -339,26 +339,112 @@ SELECT toDate('2016-12-27') AS date, toYearWeek(date) AS yearWeek0, toYearWeek(d
...
-- `minute`
-- `hour`
-- `day`
-- `week`
-- `month`
-- `quarter`
-- `year`
+## date_trunc {#date_trunc}
+
+Truncates a date or date with time based on the specified datepart.
```
Comment (BayoNet) [inline docs/en/sql-reference/functions/date-time-functions.md:344]:
```suggestion
Truncates date and time data to the specified precision.
```

File: docs/en/sql-reference/functions/date-time-functions.md [hunk 2]
```diff
@@ -339,26 +339,112 @@ SELECT toDate('2016-12-27') AS date, toYearWeek(date) AS yearWeek0, toYearWeek(d
...
+
+**Syntax** 
+
+``` sql
+date_trunc(datepart, value[, time_zone])
```
Comment (BayoNet) [inline docs/en/sql-reference/functions/date-time-functions.md:349]:
```suggestion
date_trunc(precision, value[, time_zone])
```

File: docs/en/sql-reference/functions/date-time-functions.md [hunk 3]
```diff
@@ -339,26 +339,112 @@ SELECT toDate('2016-12-27') AS date, toYearWeek(date) AS yearWeek0, toYearWeek(d
...
+```
+
+Alias: `dateTrunc`. 
+
+**Parameters**
+
+-   `datepart` — Name of the datepart. [String](../syntax.md#syntax-string-literal).
```
Comment (BayoNet) [inline docs/en/sql-reference/functions/date-time-functions.md:356]:
```suggestion
-   `precision` — Required precision. [String](../syntax.md#syntax-string-literal).
```

File: docs/en/sql-reference/functions/date-time-functions.md [hunk 4]
```diff
@@ -339,26 +339,112 @@ SELECT toDate('2016-12-27') AS date, toYearWeek(date) AS yearWeek0, toYearWeek(d
...
+    - `month`
+    - `quarter`
+    - `year`
+
+-   `value` — Date and time. [DateTime](../../sql-reference/data-types/datetime.md) or [DateTime64](../../sql-reference/data-types/datetime64.md).
+-   `timezone` — [Timezone name](../../operations/server-configuration-parameters/settings.md#server_configuration_parameters-timezone) for the returned value (optional). If not specified, the function uses the timezone of the `value` parameter. [String](../../sql-reference/data-types/string.md).
+
+**Returned value**
+
+-   Value, truncated to the specified datepart.
```
Comment (BayoNet) [inline docs/en/sql-reference/functions/date-time-functions.md:373]:
```suggestion
-   Value, truncated to the specified precision.
```

**Pull request merged**